toolkits/global/packages/global-forms/scss/50-components/_forms.scss

```diff
@@ -1,78 +1,236 @@
 @use 'sass:math';
-/* Input row component */
 
-.c-input-row {
+.c-forms__label,
+.c-forms__hint,
+.c-forms__error,
+.c-forms__legend {
+	display: block;
+}
+
+.c-forms__hint,
+.c-forms__error {
+	font-weight: normal;
+}
+
+.c-forms__input-container {
+	position: relative;
+}
+
+.c-forms__field,
+.c-forms__error-summary {
+	font-size: $global-forms-input-font-size;
+}
+
+.c-forms__field small {
+	font-size: inherit;
+}
+
+.c-forms__fieldset,
+.c-forms__legend {
+	padding: 0;
+	border: 0;
+}
+
+.c-forms__input,
+.c-forms__error-summary,
+.c-forms__input-container {
+	max-width: $global-forms-input-width-maximum;
+}
+
+:is(.c-forms__field, .c-forms__fieldset) + :is(.c-forms__field, .c-forms__fieldset) {
```

Comment:
Wow just found out about this amazing pseudo class I may have been dreaming of for years.
Is this safe to use, support-wise?

Reply:
@morgaan Come to think of it, I'm not sure our position on IE about this currently. Have rolled back for the time being. https://github.com/springernature/frontend-toolkits/pull/743/commits/28a443419b5c5b1d538ef59863c91e9f7c307d84